_includes/default-header.html

```diff
@@ -47,14 +47,14 @@
           </div>
 
           <div class="navbar-item has-dropdown is-hoverable">
-            <a class="navbar-link" href="/ols-7">OLS-7</a>
+            <a class="navbar-link" href="/openseeds/ols-7">OLS-7</a>
```

Comment:
We can probably remove OLS-7 from the navbar

Reply:
Right. 💯
Removed!